arelle/plugin/validate/EDINET/rules/edinet.py

```diff
@@ -911,6 +911,60 @@ def rule_EC8034W(
+        'http://disclosure.edinet-fsa.go.jp/role/jpcrp/rol_std_CabinetOfficeOrdinanceOnDisclosureOfCorporateInformationEtcFormNo27SecuritiesRegistrationStatement',
+        'http://disclosure.edinet-fsa.go.jp/role/jpcrp/rol_std_CabinetOfficeOrdinanceOnDisclosureOfCorporateInformationEtcFormNo3AnnualSecuritiesReport',
+        'http://disclosure.edinet-fsa.go.jp/role/jpcrp/rol_std_CabinetOfficeOrdinanceOnDisclosureOfCorporateInformationEtcFormNo32AnnualSecuritiesReport',
+        'http://disclosure.edinet-fsa.go.jp/role/jpcrp/rol_std_CabinetOfficeOrdinanceOnDisclosureOfCorporateInformationEtcFormNo4AnnualSecuritiesReport',
+        'http://disclosure.edinet-fsa.go.jp/role/jpcrp/rol_std_CabinetOfficeOrdinanceOnDisclosureOfCorporateInformationEtcFormNo43QuarterlySecuritiesReport',
+        'http://disclosure.edinet-fsa.go.jp/role/jpcrp/rol_std_CabinetOfficeOrdinanceOnDisclosureOfCorporateInformationEtcFormNo43SemiAnnualSecuritiesReport',
+        'http://disclosure.edinet-fsa.go.jp/role/jpcrp/rol_CabinetOfficeOrdinanceOnDisclosureOfCorporateInformationEtcFormNo24SecuritiesRegistrationStatement',
+        'http://disclosure.edinet-fsa.go.jp/role/jpcrp/rol_CabinetOfficeOrdinanceOnDisclosureOfCorporateInformationEtcFormNo27SecuritiesRegistrationStatement',
+        'http://disclosure.edinet-fsa.go.jp/role/jpcrp/rol_CabinetOfficeOrdinanceOnDisclosureOfCorporateInformationEtcFormNo3AnnualSecuritiesReport',
+        'http://disclosure.edinet-fsa.go.jp/role/jpcrp/rol_CabinetOfficeOrdinanceOnDisclosureOfCorporateInformationEtcFormNo32AnnualSecuritiesReport',
+        'http://disclosure.edinet-fsa.go.jp/role/jpcrp/rol_CabinetOfficeOrdinanceOnDisclosureOfCorporateInformationEtcFormNo4AnnualSecuritiesReport',
+        'http://disclosure.edinet-fsa.go.jp/role/jpcrp/rol_CabinetOfficeOrdinanceOnDisclosureOfCorporateInformationEtcFormNo43QuarterlySecuritiesReport',
+        'http://disclosure.edinet-fsa.go.jp/role/jpcrp/rol_CabinetOfficeOrdinanceOnDisclosureOfCorporateInformationEtcFormNo43SemiAnnualSecuritiesReport'
+    )
+    if not hasPresentationalConceptsWithFacts(val.modelXbrl, roleUris):
+        return
+    totalStockShares = val.modelXbrl.factsByQname.get(pluginData.issuedSharesTotalNumberOfSharesEtcQn)
+    if totalStockShares is None:
+        return
+    if (
+            val.modelXbrl.factsByQname.get(pluginData.corporateGovernanceCompanyWithAuditAndSupervisoryCommitteeTextBlockQn) is None and
+            val.modelXbrl.factsByQname.get(pluginData.corporateGovernanceCompanyWithCorporateAuditorsTextBlockQn) is None and
+            val.modelXbrl.factsByQname.get(pluginData.corporateGovernanceCompanyWithNominatingAndOtherCommitteesTextBlockQn) is None
+    ):
+        yield Validation.error(
+                    codes='EDINET.EC8069W',
+                    msg=_("If there are no problems with consistency with the text structure of the submitted documents, please tag using one of the following three elements: \n"
```

Comment:
> If there are no problems with consistency with the text structure of the submitted documents

I'm guessing this is a result of the translation, but it isn't clear what this part (and above) is trying to communicate. It seems like the "if" condition is whether or not shares info has been tagged, so maybe there's a better way to communicate that.